frontend/apps/marketing/src/components/overline/Overline.tsx

```diff
@@ -0,0 +1,58 @@
+import '@code-dot-org/component-library/typography/index.css';
+import {
+  default as Typography,
+  VisualAppearance,
+} from '@code-dot-org/component-library/typography';
+import {ComponentSizeXSToL} from '@code-dot-org/component-library/common/types';
+import classNames from 'classnames';
+import React, {ReactNode} from 'react';
+
+import moduleStyles from './overline.module.scss';
+
+type OverlineVisualAppearance = Extract<
+  VisualAppearance,
+  'overline-one' | 'overline-two' | 'overline-three'
+>;
+
+type OverlineProps = {
+  /** Overline content */
+  children: ReactNode;
+  /** Overline size */
+  size: Exclude<ComponentSizeXSToL, 'xs'>;
+  /** Overline color */
+  color: 'primary' | 'secondary';
+  /** ClassName passed by contentful to apply styles that are set through contentful native editor*/
```

Comment:
Super nit, not blocking:
```suggestion
  /** ClassName passed by Contentful to apply styles that are set through Contentful native editor*/
```